src/Swarm/TUI/View.hs

```diff
@@ -419,11 +419,12 @@ robotsListWidget s = viewport RobotsViewport Vertical (hCenter table)
   isNear robot = creative || distance (realToFrac <$> robot ^. robotLocation) basePos < 32
   robots :: [Robot]
   robots =
-    filter (\robot -> cheat || (isRelevant robot && isNear robot))
+    filter (\robot -> debugging || (isRelevant robot && isNear robot))
       . IM.elems
       $ g ^. robotMap
   creative = g ^. creativeMode
   cheat = s ^. uiState . uiCheatMode
+  debugging = creative && cheat
```

Comment:
I was going to say this should just be `creative`, but after thinking about it a minute I think I understand the point --- you don't necessarily want to see all the system robots whenever you are a normal player playing in creative mode.  `debugging = creative && cheat` makes sense to me.